search/search-client-query-transformer/src/main/java/io/camunda/search/clients/transformers/filter/UserTaskFilterTransformer.java

```diff
@@ -156,6 +173,10 @@ private SearchQuery getNameQuery(final List<String> name) {
     return stringTerms(NAME, name);
   }
 
+  private List<SearchQuery> getTagsQuery(final List<Operation<String>> tags) {
```

Comment:
🔧 this could be renamed for clarity to something like `getTagsAdvancedQueries` or so